jaxrs/src/test/scala/com/cerner/beadledom/jaxrs/exceptionmappers/ThrowableExceptionMapperSpec.scala

```diff
@@ -0,0 +1,138 @@
+package com.cerner.beadledom.jaxrs.exceptionmappers
+
+import com.cerner.beadledom.jaxrs.models.JsonError
+import com.cerner.beadledom.jaxrs.provider.{FakeRepository, FakeResource}
+
+import com.fasterxml.jackson.jaxrs.json.JacksonJsonProvider
+
+import org.jboss.resteasy.mock._
+import org.mockito.Mockito
+import org.mockito.Mockito._
+
+import play.api.libs.json.Json
+
+import javax.ws.rs.core.HttpHeaders.CONTENT_TYPE
+import javax.ws.rs.core.MediaType
+import javax.ws.rs.core.Response.Status._
+
+/**
+  * Unit tests for [[ThrowableExceptionMapper]].
+  *
+  * @author Cal Fisher
+  */
+class ThrowableExceptionMapperSpec extends BaseExceptionMapperSpec {
+
+  val throwableExceptionMapper = new ThrowableExceptionMapper
+  val fakeRepository = mock[FakeRepository]
+  val fakeResource = new FakeResource(fakeRepository)
+  val url = "/fakeResource/ExceptionEndpoint"
+  val request = MockHttpRequest.get(url)
+  var response: MockHttpResponse = _
+  val internalServerErrorJsonError = createJsonError(INTERNAL_SERVER_ERROR)
+
+  before {
+    Mockito.reset(fakeRepository)
+    response = new MockHttpResponse
+  }
+
+  describe("ThrowableExceptionMapper") {
+    describe("unit testing the behavior of ThrowableExceptionMapper class when an exception is " +
```

Comment:
Could this be shorted to something like `when exceptions are thrown from a resource`?

Reply:
I've shortened this `describe` (for all three specs): https://github.com/cerner/beadledom/pull/45/commits/e1721a2548f63bd69d56eba9dcd838de312ed2d7#diff-0734403a349f9724fd379e4bead2c7e8R47